### PR TITLE
Components

### PR DESCRIPTION
**Pull request by samuelmoiwa**
## Solution description

**React:** React app setup and configurations
**React Component:** add React components to fit the design file.
**Linters files:** linters files configured

## Address affected and ensured
* HTML
* JavaScript
* CSS file
* React config
* Linters Config
---

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
```
Comment (youmari) [inline src/App.js]: - [ ] You shouldn't delete the App component make sure to add it to the index component and it should be classed based not function then put the `Calculator.js` component in the App.js in order to match the requirements 
-Screenshot 
Screen Shot 2022-10-26 at 15 50 55

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @samuelmoiwa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (samuelmoiwa) [reply src/App.js]: hi @youmari , 
I have already refactored the project, 
I don't think I have this App.js in the new branch "components"
Screenshot 2022-10-26 150433

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (Munsa1) [inline src/index.js]: - [ ] As requested by the previous reviewer, the calculator component must be placed in `App.js`, the best you could do is create a new react app and transfer the code from the one you have to the one you will create.

image

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable react/no-unknown-property */
```
Comment (Munsa1) [inline src/components/Calculator.js]: - [ ] Kindly do not use the disable linter feature as linters are used to guide us to write industry standard code.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @samuelmoiwa, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @samuelmoiwa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**